lib/fog/vsphere/requests/compute/list_compute_resources.rb

```diff
@@ -0,0 +1,58 @@
+module Fog
+  module Compute
+    class Vsphere
+      class Real
+        def list_compute_resources(filters = { })
```

Comment:
@nirvdrum, understand, we checked the implementations like list_vm_volumes/list_datastores, they all return symbol keys.